week-13/week-13-notes.md

```diff
@@ -0,0 +1,248 @@
+# Week 13 - Self Care & Social call
+
+**Date**: December 14, 2023
+
+**Time**: 16:00
+
+**Duration**: 90 min
+
+**Call lead**: Malvika
+
+**Facilitator**: Gigi Kenneth
+
+## Join the Cohort Room
+
+**Join the Zoom call**:
+
+```         
+[https://us02web.zoom.us/j/85250504995?pwd=bGJKOHFoT0lsVENMbEZENm1rc3l0dz09](https://us02web.zoom.us/j/85250504995?pwd=bGJKOHFoT0lsVENMbEZENm1rc3l0dz09)
+```
+
+**Breakout room**: Speaking and Writing:
+
+-   Please edit your Zoom name (click on the three dots on the top right of your video) and add one of the following letters in front of your name:
+-   W for written reflection-based exercise in the main room
+-   S for Spoken Discussion Breakout Room This will help us assign you to the breakout room with the format of your choice
+-   If you are ok with both, please choose one for this week so that the hosts can assign you to a breakout room during the cohort call
+
+## 🌍 Roll call
+
+### Introducing yourself
+
+Name / Project / social handles (twitter, GitHub, etc.) / \_emoji mood \_
+
+-   Apologies from Derek Moore. Celebrating family Christmas on the 14th. Kids off to Australia
+-   Virginia García Alonso / Red JICMar / @VA\_GarciaAlonso / 🧉
+-   Monica Alonso / Metadocencia /@monicalonso2000
+-   Diana Karan/Bionformatics Codeathon/github @dianakaran
+-   Yan-Kay Ho / Reclone Community "Building a Searchable Open Data Repository of DNA Collection" / @Just\_YanKay \| github.com/ykho001 /
+-   Jafsia elisee/Mboalab
+-   Cibele Zolnier / Reclone Community "Building a Searchable Open Data Repository of DNA Collection"
+-   Malvika Sharan
+-   Meli / MetaDocencia cohortes / 🍰
+-   PaulineGachanja
```

Comment:
```suggestion

```
Deleting participant's names to anonymize the notes. We had missed this step in previous calls, but we're trying to be more careful about it.